Stripe/Payments/STPPaymentHandler.m

```diff
@@ -816,8 +823,59 @@ - (void)transactionDidPresentChallengeScreen:(STDSTransaction *)transaction {
                                                                                uiType:transaction.presentedChallengeUIType];
 }
 
+- (void)_markChallengeCanceledWithCompletion:(STPBooleanSuccessBlock)completion {
```

Comment:
I'm guessing not b/c of the PR title, but do we want to call this for native 3DS2 cancels as well?  Not obvious why we don't need to

Reply:
We call this for hosted 3DS2, but not native since native 3DS2 cancels already go to the ACS

Reply:
Could you add a comment saying something to this effect?